symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -728,6 +729,19 @@ public V1AuditTrailInitiatorList listAuditTrail(@Nonnull Long startTimestamp, Lo
             startTimestamp, endTimestamp, before, after, limit, initiatorId, role));
   }
 
+  /**
+   * Suspends or re-activates a user account.
+   *
+   * @param userId  user id
+   * @param userSuspension action to perform on the user
+   * @see <a href="https://developers.symphony.com/restapi/reference#suspend-user-v1">Suspend User Account v1</a>
+   */
+  @API(status = API.Status.STABLE)
```

Comment:
API status is already defined at class level

Reply:
We could maybe start setting the [`since` param](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html#since()) though :)

Reply:
removed! :)